python/paddle/base/variable_index.py

```diff
@@ -180,6 +180,7 @@ def deal_advanced_index(ori_tensor, indices, is_for_setitem):
         trans_back_dim (List): order of axes to transpose back to original order. Only used in __setitem__.
         pos_of_new_dim (int):  axis of new dim in the result. Only used in __getitem__.
         rank_of_new_dim (int): rank of new dim in the result. Only used in __getitem__.
+        transed_value_tensor (Tensor): value tensor transed to the front. Only used in __setitem__.
```

Comment:
should be `transposed_value_tensor`? please pay attention to it in the future

Reply:
> should be `transposed_value_tensor`? please pay attention to it in the future

ok, this would be modified in another PR